### PR TITLE
Created Users schema and CRUD endpoints

### PR DESCRIPTION
**Pull request by amyqiu**
For #68

### CODE REVIEW

File: models/User.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  school: String,
+  day: {
+    type: Number,
+    min : 1,
+    max : 5
+  },
+  phoneNumber: String
+}, {timestamps: true});
+
+// Requires population of author
```
Comment (theRoughCode) [inline models/User.js:14]: What does this comment mean?

**Review by theRoughCode — APPROVED** [review]
Great work!

**Comment by theRoughCode** [comment]
Merged PR, but left a comment regarding a comment

File: models/User.js (hunk 1 above)
Comment (amyqiu) [reply models/User.js:14]: Whoops I copied the faq schema and left this comment in by accident, not sure what it's referring to

**Pull request merged**